lib/src/helper.dart

```diff
@@ -104,9 +116,19 @@ class Adaptive {
 
   /// Calculates the width depending on the device's screen size
   ///
-  /// Eg: 20.h -> will take 20% of the screen's width
+  /// Eg: 20.w -> will take 20% of the screen's width
   static double w(num width) => width.w;
 
+  /// Calculates the height depending on the device's screen size
+  ///
+  /// Eg: 20.sh -> will take 20% of the safe area height
+  static double sh(num height) => height.h;
```

Comment:
Isn't this supposed to return `height.sh`?

Reply:
Good catch. Thank you.
I updated the PR.